docs/src/Main.tsx

```diff
@@ -86,6 +86,10 @@ const App = () => {
             <Steps />
           </Route>
 
+          <Route path="/switch">
+            <Switch />
+          </Route>
+
```

Comment:
Maybe remove the switch route from below that is commented out, line 122?🙂

Reply:
Ah of course. 😄 👍

Reply:
done